scripts/docker/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+PBUID=$(id -u pajbot)
+PBGID=$(id -g pajbot)
+
+if [[ -z "${PBUID}${PBGID}" ]]; then
```

Comment:
seems like it would be inaccurate if you had only one of the two things (e.g. pajbot user but no pajbot group, or pajbot group but no pajbot user)